client/my-sites/email/email-management/style.scss

```diff
@@ -440,3 +440,64 @@
 		}
 	}
 }
+
+.context-all-domain-management {
+	&.main {
+		width: 100%;
+	}
+
+	.navigation-header {
+		border-block-end: none !important;
+	}
+
+	.navigation-header__main {
+		padding-inline: 0 !important;
```

Comment:
I definitely don't have a full handle on the CSS here, but it seems like we could maybe tweak `client/my-sites/domains/domain-management/list/style.scss` instead of using `!important` here..?

Reply:
The css selector has been rearanged to avoid adding `!important` operator.

Reply:
AFAICT, the `!important` operator is still there. Not a requirement or anything, but I still wanted to point it out